pyproject.toml

```diff
@@ -79,7 +80,7 @@ jws = "josepy.jws:CLI.run"
 line-length = 100
 # This should be kept in sync with the version of Python specified in poetry's
 # dependencies above.
-target-version = ['py39', 'py310', 'py311', 'py312', 'py313']
+target-version = ['py39', 'py310', 'py311', 'py312', 'py313'] # TODO add 'py314'
```

Comment:
TODOs should have an associated issue tracking the plan to complete them

Reply:
Not sure that's really necessary here. I only added the comment since the current black version doesn't support `py314`. The line will also become redundant once the build-system is updated to poetry `v2` which supports `project.requires-python`.

Reply:
Those are the sorts of details that would be great in an issue so that we remember to update this code when they become available.

Reply:
Opened #232 and updated the comment here.